build/Containerfile.builder

```
@@ -15,7 +15,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM registry.access.redhat.com/ubi9/nodejs-22-minimal:9.6-1755749564
+FROM registry.access.redhat.com/ubi10/nodejs-24-minimal:10.1-1764649373
```

RuboCop (1.82.0)

[fatal] unexpected token tCOLON
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)